crates/frontend/src/pages/experiment_list/filter.rs

```diff
@@ -280,29 +291,54 @@ pub(super) fn experiment_table_filter_widget(
             handle_close=move || close_drawer("experiment_filter_drawer")
         >
             <div class="flex flex-col gap-5">
-                <ContextForm
-                    dimensions=dim
-                    context=context_rws.get_untracked()
-                    on_context_change=move |context: Conditions| {
-                        context_rws.set(context.clone());
-                        let map = context
-                            .iter()
-                            .filter_map(|condition| {
-                                match condition.expression.clone() {
-                                    Expression::Is(value) => {
-                                        Some((condition.variable.clone(), value))
-                                    }
-                                    _ => None,
-                                }
-                            })
-                            .collect::<Map<_, _>>();
-                        dimension_buffer_rws.set(DimensionQuery::from(map));
+                {move || {
+                    view! {
+                        <ContextForm
+                            dimensions=combined_resource.dimensions.clone()
+                            context=context_rws.get_untracked()
+                            on_context_change=move |context: Conditions| {
+                                context_rws.set(context.clone());
+                                let map = context
+                                    .iter()
+                                    .filter_map(|condition| {
+                                        match condition.expression.clone() {
+                                            Expression::Is(value) => {
+                                                Some((condition.variable.clone(), value))
+                                            }
+                                            _ => None,
+                                        }
+                                    })
+                                    .collect::<Map<_, _>>();
+                                dimension_buffer_rws.set(DimensionQuery::from(map));
+                            }
+                            heading_sub_text="Search By Context"
+                            resolve_mode=true
+                            fn_environment
+                            disabled=filters_buffer_rws
+                                .with(|f| f.global_experiments_only.unwrap_or_default())
+                        />
                     }
-                    heading_sub_text="Search By Context"
-                    resolve_mode=true
-                    fn_environment
-                />
-                <div class="w-full flex flex-row justify-start items-end gap-10">
+                }} <div class="w-fit flex items-center gap-2">
```

Comment:
```suggestion
                }} 
                <div class="w-fit flex items-center gap-2">
```
Has this been formatted this way by leptosfmt?